src/detect-flowbits.c

```diff
@@ -334,7 +238,7 @@ int DetectFlowbitSetup (DetectEngineCtx *de_ctx, Signature *s, const char *rawst
     sm->type = DETECT_FLOWBITS;
     sm->ctx = (SigMatchCtx *)cd;
```

Comment:
Q1: If I were to make the struct opaque to C, how would I set sigmatch context here?
Q2: SigMatchCtx seems to have just one member `int foo`,  does it matter what i assign to it?
Q3: What part of the code should I look at to understand the importance of context?

Reply:
I think it might be helpful to check how `DetectHTTP2sizeUpdateSetup` and the functions below it handle this. Essentially from C there is only a void pointer.